tests/reference/ast_new-string1-96b90b3.stdout

```diff
@@ -499,20 +499,20 @@
     (Expr
         (JoinedStr
             [(ConstantStr
-                "\\n"
+                "\n"
                 ()
             )]
         )
     )
     (Expr
         (ConstantBytes
-            "b'\n'"
+            "b'\\n'"
             ()
         )
     )
     (Expr
         (ConstantBytes
-            "b'\n\n\\n'"
+            "b'\n\\n\\\\n'"
```

Comment:
In the example above, the given string is
```
b'''
\n\\n'''
```

With respect to this example, it seems that all strings (including raw strings, bytes, raw bytes, etc.) need unescaping at the tokenizer level. And then we need to escape them if they are raw strings/ bytes/ raw bytes, etc (I think at the parser level).

Reply:
Why do raw strings have to be escaped?

Reply:
I think you are right, Sir. I will look into it.